pkgs/top-level/all-packages.nix

```diff
@@ -22460,6 +22460,17 @@ with pkgs;
 
   libavif = callPackage ../development/libraries/libavif { };
 
+  libavif_0_11 = libavif.overrideAttrs rec {
```

Comment:
BTW, I don't like this approach of adding a named version only for one package (if there is more, this would be fine).

Can you overwrite `libavif` only for `ansel`?

Reply:
done, also added a comment explaining why it's needed.